lib/api/github.js

```diff
@@ -59,8 +59,10 @@ async function initRepo(repoName, token, endpoint) {
 async function findFilePaths(fileName) {
   const res = await ghGot(`search/code?q=repo:${config.repoName}+filename:${fileName}`);
   const exactMatches = res.body.items.filter(item => item.name === fileName);
-  const filePaths = exactMatches.map(item => item.path);
-  return filePaths;
+
+  // GitHub seems to return files in the root with a leading `/`
+  // which then breaks things later on down the line
+  return exactMatches.map(item => item.path.replace(/^\//, ''));
 }
```

Comment:
@jonbretman I would like to make sure I understand your problem properly so that I can confirm that this is the right fix. I've only been using the default endpoint for GitHub myself.

I'm guessing you are using a custom endpoint, and without this code change it was failing? Did this fail regardless of whether your custom endpoint had a trailing slash or not? e.g. `https://mycustomgithubendpoint.com/api/` versus `https://mycustomgithubendpoint.com/api`?

BTW we use library `gh-got` and they're doing their own URL manipulation here: https://github.com/sindresorhus/gh-got/blob/master/index.js#L15